paymentsheet/src/main/java/com/stripe/android/link/theme/Color.kt

```diff
@@ -72,62 +59,104 @@ internal object LinkThemeConfig {
-        secondaryButtonLabel = LightSecondaryButtonLabel,
-        sheetScrim = LightSheetScrim,
-        linkLogo = LightLinkLogo,
-        closeButton = LightCloseButton,
-        disabledText = LightTextDisabled,
-        otpElementColors = OTPElementColors(
-            selectedBorder = LinkTeal,
-            placeholder = LightOtpPlaceholder
-        ),
-        actionLabel = ActionGreen,
+        surfacePrimary = Neutral0,
+        surfaceSecondary = Neutral100,
+        surfaceTertiary = Neutral200,
+        surfaceBackdrop = Neutral900,
+        borderDefault = Neutral300,
+        borderSelected = Neutral900,
+        borderCritical = Critical500,
+        buttonPrimary = Neutral900,
+        buttonSecondary = Neutral100,
+        buttonTertiary = Neutral0,
+        buttonBrand = Brand200,
+        buttonCritical = Critical500,
+        textPrimary = Neutral900,
+        textSecondary = Neutral700,
+        textTertiary = Neutral500,
+        textWhite = Neutral0,
+        textBrand = Brand600,
+        textCritical = Critical600,
+        iconPrimary = Neutral900,
+        iconSecondary = Neutral700,
+        iconTertiary = Neutral500,
+        iconWhite = Neutral0,
+        iconBrand = Brand200,
+        iconCritical = Critical500
     )
 
-    private val colorsDark = colorsLight.copy(
-        primary = LinkTeal,
-        secondary = DarkFill,
-        background = DarkBackground,
-        textPrimary = DarkTextPrimary,
-        textSecondary = DarkTextSecondary,
-        componentBackground = DarkComponentBackground,
-        componentBorder = DarkComponentBorder,
-        componentDivider = DarkComponentDivider,
-        progressIndicator = DarkProgressIndicator,
-        linkLogo = DarkLinkLogo,
-        closeButton = DarkCloseButton,
-        disabledText = DarkTextDisabled,
-        secondaryButtonLabel = DarkSecondaryButtonLabel,
-        otpElementColors = OTPElementColors(
-            selectedBorder = LinkTeal,
-            placeholder = DarkOtpPlaceholder
-        ),
+    private val colorsDark = LinkColors(
+        surfacePrimary = Neutral900,
+        surfaceSecondary = Neutral800,
+        surfaceTertiary = Neutral700,
+        surfaceBackdrop = Neutral900,
+        borderDefault = Neutral900,
+        borderSelected = Brand200,
+        borderCritical = Critical500,
+        buttonPrimary = Neutral200,
+        buttonSecondary = Neutral700,
+        buttonTertiary = Neutral800,
+        buttonBrand = Brand200,
+        buttonCritical = Critical600,
+        textPrimary = Neutral0,
+        textSecondary = Neutral300,
+        textTertiary = Neutral500,
+        textWhite = Neutral0,
+        textBrand = Brand200,
+        textCritical = Critical600,
+        iconPrimary = Neutral100,
+        iconSecondary = Neutral500,
+        iconTertiary = Neutral500,
+        iconWhite = Neutral0,
+        iconBrand = Brand200,
+        iconCritical = Critical500
     )
+
+    /**
```

Comment:
Instead of exposing colors directly across Views, unifying non pure Link Design systems workarounds here. That should help iterating in the future as we get closer to pure Link designs.